src/app/models/Tag.ts

```diff
@@ -26,36 +26,28 @@ export interface TagInterface {
 /**
  * Tag model
  */
-export class Tag extends AbstractModel implements TagInterface {
+export class Tag extends AbstractModel implements ITag {
   public readonly kind: "AbstractModel" = "AbstractModel";
   public readonly id?: Id;
   public readonly text?: string;
   public readonly count?: number;
   public readonly isTaxanomic?: boolean;
   public readonly typeOfTag?: string;
```

Comment:
https://github.com/QutEcoacoustics/baw-server/blob/d165c88677ac97f1fa6be47d28b7774b1bed7846/app/models/tag.rb#L20-L26

Reply:
We agreed previously that the type of tags may be changed in the future and may become an api request. This is because of the `typeofTag`s is used as a dropdown field in a couple places.